package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "root",
   "private": true,
-  "version": "1.0.0-preview.1",
   "scripts": {
```

Comment:
Is this version removed on purpose? Should it be "1.0.0-preview.2"?

Reply:
yes this package.json isn't published so I removed the version to remove some dead code